### ISSUE TITLE
French 'Make a request' text is running over out of green box on authority pages

### ISSUE THREAD

**Issue by StephenAbbott** [mysociety/sobanukirwa-theme#69]
I spotted the following issue using Safari mobile browser on an iPhone 5:

![img_4499](https://cloud.githubusercontent.com/assets/1506468/5891421/451b284a-a49d-11e4-84e3-ea1df788285c.PNG)

The Make a request button on sobanukirwa.rw/fr homepage is fine.

Green text box on mobile version of user profile pages also seems to be extending off the right-hand side of the screen: 
![img_4502](https://cloud.githubusercontent.com/assets/1506468/5891443/a31fb8ec-a49e-11e4-97be-ac8920ed115d.PNG)

**Comment by garethrees** [comment]
**Note to self**

Fix

```css
input[type=submit], a.link_button_green, a.link_button_green_large {
text-decoration: none;
color: #fff;
background-color: #62973b;
display: inline-block;
cursor: pointer;
border-style: solid;
border-color: #4a722d;
border-width: 0 0 4px 0;
border-radius: 0;
padding: 3px 7px;
line-height: normal;
font-size: 1.1rem;
/* height: 2.3125rem; Remove fixed height on mobile. Keep on desktop otherwise the button is too thin*/
vertical-align: top;
}
```

**Comment by garethrees** [comment]
Fixed in ce52ffe
